doc/content/getting-started/migrating-from-v2/major-changes.md

```diff
@@ -13,7 +13,7 @@ Before getting started, we will discuss major breaking changes between {{% ttnv2
 
 ## LoRaWAN support
 
-{{% tts %}} requires the LoRaWAN version and Regional Parameters (LoRaWAN PHY version) to be set per end device. These default to LoRaWAN version 1.0.2 and LoRaWAN Regional Parameters version 1.0.2 Rev B for end devices imported from {{% ttnv2 %}}, because this configuration is the most consistent with the V2.
+{{% tts %}} requires the **LoRaWAN version** and **Regional Parameters** (LoRaWAN PHY version) to be set per end device. These default to LoRaWAN version `1.0.2` and LoRaWAN Regional Parameters version `1.0.2 Rev B` for end devices imported from {{% ttnv2 %}}, because this configuration is the most consistent with the V2.
```

Comment:
cc: @johanstokking for the usage of backticks

Reply:
It's not code so no backticks